pom.xml

```diff
@@ -28,6 +30,49 @@
       <url>https://github.com/thelastpickle/cassandra-reaper/issues</url>
     </issueManagement>
 
+    <distributionManagement>
+      <repository>
+        <id>ossrh</id>
+        <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+      </repository>
+      <snapshotRepository>
+        <id>ossrh</id>
+        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      </snapshotRepository>
+    </distributionManagement>
+
+  <developers>
+    <developer>
+      <id>mck</id>
+      <name>Michael Semb Wever</name>
+      <email>mick@thelastpickle.com</email>
+      <url>http://about.me/wever</url>
+    </developer>
+    <developer>
+      <id>adejanovski</id>
+      <name>Alexander Dejanovski</name>
+      <email>adejanovski@gmail.com</email>
+    </developer>
+    <developer>
+      <id>rustyrazorblade</id>
+      <name>Jon Haddad</name>
+      <url>http://www.rustyrazorblade.com/</url>
+    </developer>
+    <developer>
+      <id>joaquincasares</id>
+      <name>Joaquin Casares</name>
+      <email>joaquin.casares@gmail.com</email>
+    </developer>
+  </developers>
+
+  <contributors>
+    <contributor>
+      <name>Marvin Herman Froeder</name>
+      <email>velo br at gmail dot com</email>
+      <url>about.me/velo</url>
+    </contributor>
+  </contributors>
```

Comment:
i'd take away developers and contributors section. is it needed for rultor?

this information is stale and superfluous. live versions of it can be found via the changelog or github insights.

Reply:
`developers` is required to publish on central
http://central.sonatype.org/pages/requirements.html#developer-information

contributors not.

Reply:
Ok.

Then can we add some of the previous developers, so it doesn't look like TLP has completely taken over.